Homework7/Book.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace Homework7
+{
+    interface IReadable 
+    {
+        string? Description { get; }
+        void TurnPage();
+    }
+    interface IRecyclable
+    {
+        void Recycle();
+    }
+    internal class Book:IReadable, IRecyclable
+    {
+        public string? Description { get { return "This is usual book."; } }
+        public void TurnPage()
+        {
+            Console.WriteLine("Page turn sound...");
+        }
+        public void Recycle()
+        {
+            Console.WriteLine("Sound of fire crackling...");
+        }
+    }
+    internal class EBook : IReadable, IRecyclable
+    {
+        public string? Description { get { return "This is electronic book."; } }
```

Comment:
You can replace with lambda expression:
public string? Description => "This is electronic book."; (it will be readonly property)

Reply:
fixed